lib/chunk_downloader.c

```diff
@@ -233,7 +233,6 @@ SF_CHUNK_DOWNLOADER *STDCALL chunk_downloader_init(const char *qrmk,
     // We need thread_count, fetch_slots, chunks, and either qrmk or chunk_headers
     if (thread_count <= 0 ||
             fetch_slots <= 0 ||
-            !(chunk_headers || qrmk) ||
```

Comment:
Large result set functionality in Azure does not require either chunk_headers or qrmk to be set